src/ra.app.src

```diff
@@ -1,6 +1,6 @@
 {application,ra,
              [{description,"Raft library"},
-              {vsn,"2.0.1"},
+              {vsn,"2.0.2"},
```

Comment:
Should this be `2.0.3` since `2.0.2` has already shipped?

Reply:
No for some reason this commit went into the wrong branch. Main has 2.0.1